frame/proxy/src/lib.rs

```diff
@@ -752,32 +772,3 @@ impl<T: Config> Pallet<T> {
 		Self::deposit_event(Event::ProxyExecuted(e.map(|_| ()).map_err(|e| e.error)));
 	}
 }
-
-/// Migration utilities for upgrading the Proxy pallet between its different versions.
-pub mod migration {
```

Comment:
Tagged as `E1-runtimemigration` because you remove the migration here.
Would prefer to remove it in a separate PR so it's easier to isolate.

Reply:
have to remove it because the changes to bounded vec are affected here.